.github/workflows/main.yml

```diff
@@ -1,14 +1,19 @@
----

```

Comment:
Why delete this line?

Reply:
This line is useless

Reply:
I disagree with that.

It's a convention to start YML files with `---`, so I would not say it's "useless"